src/component/Header/MenuBar/NavItem.jsx

```diff
@@ -41,17 +41,21 @@ const NavItem = ({ title, subTitle, subTitle1, subTitle2 }) => {
           horizontal: 'left'
         }}
       >
-        <MenuItem onClick={handleClose}>{subTitle}</MenuItem>
-        <MenuItem onClick={handleClose}>{subTitle1}</MenuItem>
-        <MenuItem onClick={handleClose}>{subTitle2}</MenuItem>
+        {subItems.map((subItem) => (
```

Comment:
you dont need to declare a keyValue const to set the key. the map function (along with most array functions) can also receive the index and you can just use it as the key